src/cpp/ext/otel/otel_server_call_tracer.cc

```diff
@@ -148,6 +167,11 @@ class OpenTelemetryServerCallTracer : public grpc_core::ServerCallTracer {
   grpc_core::Slice path_;
   std::unique_ptr<LabelsIterable> injected_labels_;
   bool registered_method_;
+  std::vector<bool>
```

Comment:
Same comment as elsewhere: We can probably save memory by using a bitmask here.

Reply:
Suggestion taken